cms/djangoapps/contentstore/rest_api/v2/urls.py

```diff
@@ -23,6 +24,16 @@
         downstreams.DownstreamView.as_view(),
         name="downstream"
     ),
+    re_path(
+        f'^upstreams/{settings.COURSE_KEY_PATTERN}$',
+        downstreams.UpstreamListView.as_view(),
+        name='upstream-list'
+    ),
+    re_path(
+        f'^upstream/{settings.USAGE_KEY_PATTERN}/downstream-contexts$',
```

Comment:
I'm not happy with my choice of URL here. I was trying to have a structure like `parent/{parentId}/children` instead of `children/{parentId}`, but I`m not sure if this is clear close to the other endpoints.

Could you think you have a better idea?